src/locales/en-US.json

```diff
@@ -451,7 +451,7 @@
             "sent": "Sent",
             "received": "Received",
             "this_address": "this address",
-            "invoke_function": "INVOKED FUNCTION",
+            "invoke_function": "Sent",
```

Comment:
There is genuinely a scenario when "INVOKED FUNCTION" is appropriate. We should create a new entry for "Sent" and use it specifically for sent transactions.